.github/workflows/openapi-linter.yaml

```diff
@@ -0,0 +1,15 @@
+name: Lint OpenAPI
```

Comment:
Anything that we want to avoid exposing to online swagger editor that indicate that we should use local dockerized validation instead as described here? https://github.com/char0n/swagger-editor-validate

Reply:
I belive only nva-infrastructure is closed source, and not already public information. But using a docker image might be more stable in the long run:

- Our builds wont fail if swagger.io is down.
- We can version lock the image so we dont risk breaking on new versions